### PR TITLE
Dockerfile added and readme updated

### PR DESCRIPTION
**Pull request by kevinassogba**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+COPY /server /github-graphs/server
+COPY README.md /github-graphs/
+COPY LICENSE /github-graphs/
+
+# setup working directory to the default in the container
+WORKDIR /github-graphs/server
+
+# Install dependencies and start the program at RUN
+RUN npm install
+ENTRYPOINT ["node", "server.js"]
```
Comment (Tjzabel) [inline Dockerfile]: I think you're looking for the `CMD` command here. `ENTRYPOINT` usually points to an actual script file before the `CMD`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM ubuntu
+MAINTAINER Jeffery Russell
```
Comment (Tjzabel) [inline Dockerfile]: This format is actually slightly outdated. The correct labeling would be:
```
LABEL maintainer="Jeffery Russell"
```

**Review by Tjzabel — CHANGES_REQUESTED** [review]
Looks good! Just a few nitpicks :+1:

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+services:
+    github-graphs:
+        build:
+            context: .
+            dockerfile: Dockerfile
+        image: graph-app:latest
+        container_name: github-graphs
+        ports:
+            - "8080:8000"
+        command: /bin/bash
```
Comment (Tjzabel) [inline docker-compose.yml]: `/bin/bash` shouldn't be necessary to run here

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+version: '3.7'
+
+services:
+    github-graphs:
+        build:
+            context: .
+            dockerfile: Dockerfile
```
Comment (Tjzabel) [inline docker-compose.yml:7]: This is fine, but in general if the context is the same directory, you don't need to give context, as it looks for a `Dockerfile` in the root by default.

File: Dockerfile (hunk 1 above)
Comment (kevinassogba) [reply Dockerfile]: Thanks for the remark.

**Review by jrtechs — APPROVED** [review]
@10eMyrT Looks great! Thanks for working on this

**Pull request merged**